packages/vitest-environment-nuxt/package.json

```diff
@@ -60,5 +59,10 @@
     "vitest": "^0.24.5 || ^0.26.0 || ^0.27.0 || ^0.28.0 || ^0.29.0 || ^0.30.0",
     "vue": "^3.2.45",
     "vue-router": "^4.0.0"
+  },
+  "devDependencies": {
```

Comment:
This duplicates the `devDependencies` 👆 

I think this might have caused some of the changes in the `pnpm-lock` file.